tasks/page/makeReadme.js

```diff
@@ -56,3 +57,7 @@ if (require.main !== module) {
 function escapeRegExp(string) {
   return string.replace(/[.*+?^${}()|[\]\\]/g, '\\$&') // $& means the whole matched string
 }
+
+function formatMarkdownCode(string) {
```

Comment:
I think this function should be called something more descriptive or pull in the other formatting step from above (in which case we need a comment to explain what the regex does).